__tests__/integration/one-app.spec.js

```diff
@@ -700,30 +747,71 @@ describe('Tests that require Docker setup', () => {
+          test('fails to get external `react-intl` for child module as an unsupplied `requiredExternal` - logs failure', async () => {
+            const requiredExternalsErrorMatch = /Failed to get external react-intl from root module/;
+
+            const requiredExternalsError = searchForNextLogMatch(requiredExternalsErrorMatch);
             await addModuleToModuleMap({
               moduleName,
               version,
             });
             // not ideal but need to wait for app to poll;
-
             await waitFor(5000);
-          });
 
-          afterAll(() => {
-            writeModuleMap(originalModuleMap);
+            await expect(requiredExternalsError).resolves.toMatch(requiredExternalsErrorMatch);
           });
 
-          test('fails to get external `react-intl` for child module as an unsupplied `requiredExternal`', async () => {
-            await expect(requiredExternalsError).resolves.toMatch(requiredExternalsErrorMatch);
+          test('fails to get external `react-intl` for child module as an unsupplied `requiredExternal` - Logs reverting message', async () => {
+            const revertErrorMatch = /There was an error loading module (?<moduleName>.*) at (?<url>.*). Reverting back to (?<workingModule>.*)/;
+            const requiredExternalsError = searchForNextLogMatch(revertErrorMatch);
+            await addModuleToModuleMap({
+              moduleName,
+              version,
+            });
+            // not ideal but need to wait for app to poll;
+            await waitFor(5000);
+            const loggedError = await requiredExternalsError;
+            const [,
+              problemModule,
+              problemModuleUrl,
+              workingUrl,
+            ] = revertErrorMatch.exec(loggedError);
+            const gitSha = await retrieveGitSha();
+            await expect(requiredExternalsError).resolves.toMatch(revertErrorMatch);
+            expect(problemModule).toBe('cultured-frankie');
+            expect(problemModuleUrl).toBe(`${testCdnUrl}/${gitSha}/${moduleName}/${version}/${moduleName}.node.js`);
+            // eslint-disable-next-line no-useless-escape
+            expect(workingUrl).toBe(`${testCdnUrl}/${gitSha}/${moduleName}/0.0.0/${moduleName}.node.js\"}`);
+          });
+          test('fails to get external `semver` for child module as an unsupplied `requiredExternal` for new module in mooduleMap', async () => {
+            const revertErrorMatch = /There was an error loading module (?<moduleName>.*) at (?<url>.*). Ignoring (?<ignoredModule>.*) until .*/;
+            const requiredExternalsError = searchForNextLogMatch(revertErrorMatch);
+            const modName = 'unhealthy-frank';
+            const modVersion = '0.0.0';
+            await addModuleToModuleMap({
+              moduleName: modName,
+              version: modVersion,
+            });
+            // not ideal but need to wait for app to poll;
```

Comment:
Please set a lower max poll time instead of waiting a full 5s

Reply:
https://github.com/americanexpress/one-app/blob/main/src/server/utils/pollModuleMap.js#L35-L40

Reply:
also for future reference here is where we set it - https://github.com/americanexpress/one-app/blob/main/prod-sample/one-app/base.env#L14